### PR TITLE
Scroll Effect Added

### PR DESCRIPTION
**Pull request by PR4NJ41**
This Fixes the issue #1
Also Fixed the image size on Hompage

### CODE REVIEW

**Comment by PR4NJ41** [comment]
[**Here**](https://drive.google.com/file/d/152Y2Ula-N3o18FsNgUcIaQOUuvsf_s-Y/view?usp=sharing) is the preview of Changes that have been made

**Comment by PR4NJ41** [comment]
[**Here**](https://drive.google.com/file/d/153-b0oqHXrQ72YQsuVUrUUjG0IqevMOd/view?usp=sharing) is the preview of Changes that have been made

File: public/index.html [hunk 1]
```diff
@@ -1,32 +1,46 @@
...
-    <meta
-      name="description"
-      content="Web site created using create-react-app"
-    />
+    <meta name="description" content="Web site created using create-react-app" />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
-    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.4.1/semantic.min.css" integrity="sha512-8bHTC73gkZ7rZ7vpqUQThUDhqcNFyYi2xgDgPDHc+GXVGHXq+xPjynxIopALmOPqzo9JZj0k6OqqewdGO3EsrQ==" crossorigin="anonymous" />
-    <link 
-    rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous" 
+    <link
```
Comment (somesh202) [inline public/index.html:11]: Are  these changes caused due to eslint?

File: public/index.html [hunk 2]
```diff
@@ -1,32 +1,46 @@
...
+      href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700&display=swap"
+    />
+    <script src="https://code.iconify.design/2/2.1.0/iconify.min.js"></script>
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
 
     <title>FMC Weekend</title>
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+    <script src="https://unpkg.com/aos@next/dist/aos.js"></script>
```
Comment (somesh202) [inline public/index.html:40]: I'd suggesr using npm packages instead of using CDN just beacuse it offers better dependency management

**Review by somesh202 — APPROVED** [review]
Rest looks good to me.

**Pull request closed without merging**